pkg/common/cns-lib/vsphere/hostsystem.go

```diff
@@ -51,7 +51,7 @@ func (host *HostSystem) GetAllAccessibleDatastores(ctx context.Context) ([]*Data
 
 	var dsMoList []mo.Datastore
 	pc := property.DefaultCollector(host.Client())
-	properties := []string{"info"}
+	properties := []string{"info", "customValue"}
```

Comment:
use common const `DatastoreInfoProperty` and define common const for `customValue` property.